corehq/form_processor/models.py

```diff
@@ -308,7 +309,8 @@ class XFormPhoneMetadata(jsonobject.JsonObject):
     location = GeoPointProperty()
 
 
-class CommCareCaseSQL(PreSaveHashableMixin, models.Model, RedisLockableMixIn, AttachmentMixin, AbstractCommCareCase):
+class CommCareCaseSQL(PreSaveHashableMixin, models.Model, RedisLockableMixIn,
+                      AttachmentMixin, AbstractCommCareCase, TrackRelatedChanges):
```

Comment:
Yeah it's unclear to me how to resolve that issue. i like that most of them are mixins and don't inherit from more templates